### PR TITLE
Add a section about AI/timescale vector

### PR DESCRIPTION
**Pull request by cevian**
# Description

Add a section for AI / timescale vector

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by jamessewell — APPROVED** [review]
Looks good @cevian!

Please note I did not test out the code snippets, have they been double checked?

**Comment by cevian** [comment]
@jamessewell yes all the code has been at least double-checked :)

File: ai/python-interface.md [hunk 1]
```diff
@@ -42,23 +42,23 @@ import uuid
...
-Load up your PostgreSQL credentials, the dafest way is with a `.env` file:
+Load up your PostgreSQL credentials, the safest way is with a `.env` file:
 
 ``` python
 _ = load_dotenv(find_dotenv(), override=True)
 service_url  = os.environ['TIMESCALE_SERVICE_URL']
 ```
 
-Next, create the client. In this tutorial, we will use the sync client. But we have an async client as well (with an identical interface that
+Next, create the client. This tutorial, uses the sync client. But the library have an async client as well (with an identical interface that
```
Comment (jgpruitt) [inline ai/python-interface.md]: "the library have" should be "the library has"

File: ai/python-interface.md [hunk 2]
```diff
@@ -109,11 +109,11 @@ vec.search([1.0, 9.0])
       array([1. , 1.3], dtype=float32),
       0.14489260377438218]]
 
-There are many search options which we will cover below in the
+There are many search options which are be covered below in the
```
Comment (jgpruitt) [inline ai/python-interface.md]: "which are be covered" should be "which are covered"

**Review by jgpruitt — CHANGES_REQUESTED** [review]

**Review by jamessewell — APPROVED** [review]
👊

**Review by jgpruitt — APPROVED** [review]

**Pull request merged**